### DISCUSSION TITLE
Change Cursor Style

### DISCUSSION THREAD

**Discussion by nadinator** [ohmyzsh/ohmyzsh#11836] [Help] [answered]
How can I change the cursor style from a block to a line?

**Comment by gabriel-lima96** [comment, marked as answer]
You can probably change it in your terminal profile/style

<img width="665" alt="image" src="https://github.com/ohmyzsh/ohmyzsh/assets/36959874/ff822fa6-646e-4f3f-829a-fa3e1e87a601">
